source/common/fileName.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+import moment from 'moment';
+
+export const filenameWithTimestamp = (prefix:string = 'logs', filetype:string = 'zip') =>
+  `${prefix}-${moment.utc().format('YYYY-MM-DDThhmmss.0SSS')}.${filetype}`;
```

Comment:
@daniloprates please use the following format `YYYY-MM-DDTHH:mm:ss.0SSS` here (and this should be reflected on the `getFilenameWithTimestamp` function.

Reply:
Oh one thing I forgot - please append `Z` to the end of the timestamp so that it looks like this: `logs-2018-07-04T08:48:19.0182Z.zip`
Thanks! :)

Reply:
Hey @nikolaglumac!

I'll add the 'Z', however, it's not possible to add the colons, as it's an invalid character for filenames.

Reply:
✔︎